### DISCUSSION TITLE
一种有偏误的想法 | /posts/error-thought/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#620] [Announcements]
在我的脑子里存在一些有偏误的思维方式，如若偶然遇上适当的契机，将有机会将它们捉出来，或再辨析，或再接受，或再修正。

https://yuanfan.rbind.io/posts/error-thought/

**Comment by HushWay** [comment]
也许因为评分很高，这几部电影/电视剧我也都看过，过年时还补看了《俗女养成记2》。《我们与恶的距离》应思悦这个角色也很让我印象深刻。具体细节我记不清了，她为了为了保护饱受舆论压力的杀人犯妹妹李大芝与精神分裂的弟弟，付出了非常大的代价，开的店被砸没有生意，未婚夫也逃跑了。我真的很佩服她为了心中的良知与世界为敌的勇气。现实中我就是很懦弱、很怂的性格，甚至不敢与别人发生争吵。孔子说：乡愿，德之贼也。我也不喜欢自己这种老好人形象，但总是顾虑太多，不愿意得罪别人。看了你的博客了解了张秋子老师，一个访谈中她说：

> 文学很少会写弱者的抵抗，它会更加直接地去写强硬的对抗。但是文学之所以敢这么写，是因为它是虚构，它不需要真的对抗了以后付出什么代价，但是人不能去学，学了以后你要付出的代价是真实的。
文学有时候之所以敢去塑造这些孤勇者，是因为它的这种道德（勇气）的实现并不需要付出实际的代价，像卡夫卡小说里，一个女孩去对抗一村子人的沉浮，在现实生活中要遭到多少的利益打压，生命安全的（威胁），但在小说里面你不用去面对，合上书这个人就终结了。

我想，一味地妥协与一味地对抗都不是好事，问题是如何才能把握好度呢？

**Reply by earfanfan** [reply]
我也很喜欢应思悦，要是真得在现实中碰到这样的人，我非常愿意和她做朋友。

>我想，一味地妥协与一味地对抗都不是好事，问题是如何才能把握好度呢？

这个问题我想了1小时43分钟也没想出来什么好的结论，最终想到的仅仅只是，具体问题具体分析。

征得我堂妹依依的同意后，我给你讲一讲她的经历哈。她去年参加工作后，部门领导总是让她们加班干活，她一直在反抗，比如在加班时间“请假”、去找领导表达意见等等，那时候她想的是要是大家都联合起来一起反抗的话，就不会是所有人都被公司压榨的结果了。她坚持了9个月后终于忍无可忍离职了，我问她如果下次还遇到这种情况怎么处理，她的答案变成了，因为就她反抗被打压给了最差的绩效，那么下一次就沉默干活攒钱，然后悄悄跳出火坑。依依遇到的这种情况，属于是她一个人的力量对抗不了整个公司的压榨，只好想办法保全自己。不过在依依前公司的一个下属分公司存在另一种情况，一整个部门的人一起反抗，最后妥协的就是公司了。

而我平时在公司内部会遇到一种情况，极个别男性很喜欢用自己的审美标准或者其他人生赢家标准去评判其他女性，超没边界感的说。之前我就观察到有个女孩每次被说都会笑，于是这个女孩就总是被说，似乎这个女孩被那么评价只是一件十分寻常的事，抑或只是开玩笑。终于有一天，这样的事落到我头上了，我也没用什么激烈的言辞反抗，就是说我跟他们不是生活在一个世界的人，再后来前面一起说过我的几个男性其中一个又在某一次说我“一身反骨”，我那次反而笑呵呵地说“你说吧，我很喜欢别人这样夸我”。后来我想过很多次，要是别人说话恶心到我了，但是别人自己完全意识不到的话，我该如何面对，反复思考过很多次以后得到的结论是，首先一定要表明态度设立边界，不然我也会像观察到的那几个女孩一样笑着回应的后果就是继续不被尊重，其次，已所不欲勿施于人，我不想要怀着恶意报复别人，最后，让这样的事就像一阵风一样吹过去吧，反正那些人不会是我的朋友，不必在意。

至于这位朋友你提到的“总是顾虑太多不愿意得罪别人“，从我的视角看，你的“不愿意得罪别人”会不会其实是“不愿意伤害别人”或者“不愿意与人起冲突”，我不了解你的具体情况，只是有一种强烈的感觉，希望你在看待自己之前不要先用什么世俗社会的标准来审判自己，如果你已经“审判了自己”，先试着换个角度“为自己脱罪”。其实我觉得懂得自省是一件好事，但同时自省总是避免不了把自己拿去与其他人比较，或者将自己放置到某一种社会标准去评判，无论在哪一种标准下发现了自己的什么不足，都不应该轻易否定自己的个人价值。因为自己的存在就是存在啊，如此独一无二。

对了哦，我觉得应思悦的未婚夫逃跑了并不算是她付出的代价，反而是她通过一些事得以看清楚那个男人的真面目。我印象比较深刻的是思悦和未婚夫在一次争吵中，那个未婚夫说他看上她是因为她总是去看他奶奶，觉得她能照顾老人应该会是个好妻子，而思悦说她去看望他奶奶并不是为了讨他欢心，只是因为看不惯他们一家对待奶奶的某种嘴脸，她愿意去看奶奶才去看奶奶的。这里就能看出来未婚夫考虑结婚时是有些利益考量的，而思悦之前估计就是发自内心地觉得未婚夫人好吧，两人思想观念本就不同。

**Comment by HushWay** [comment]
很感谢你耐心的回复！对我帮助很大！

依依的例子表示同情，觉得有囚徒困境的感觉。其实不同的人的离职成本不同（取决于心态、能力与欲望），难免有顾虑，做出不同的选择。不过也正因如此，不计个人得失的集体抵抗令人感动。

第二个例子让我想起我的小学，转过两次校，每次去新学校前我妈告诫我：如果有人欺负你，一定要还手，不然他会觉得你好欺负。还真如此，两次开学都和别人打了一架，第一次是和一个高年级，一边哭一边抓住他的腿反击。第二次没有完全动手，拉扯几下陷入僵持，他可能被我决不服软的眼神镇住了，不了了之。我的性格沉闷不合群，总有人觉得好欺负，以牙还牙这条准则保护了我。

> 希望你在看待自己之前不要先用什么世俗社会的标准来审判自己，如果你已经“审判了自己”，先试着换个角度“为自己脱罪”。

谢谢你的开解，我需要“律师”。（PS：其实我也有很多刺，只是喜欢藏着，有时候反而扎到了亲密的人，好吧，我又在审判了）

> 对了哦，我觉得应思悦的未婚夫逃跑了并不算是她付出的代价，反而是她通过一些事得以看清楚那个男人的真面目。

赞同，患难见真情，结婚了矛盾更多。

**Reply by earfanfan** [reply]
>PS：其实我也有很多刺，只是喜欢藏着，有时候反而扎到了亲密的人，好吧，我又在审判了

哈哈哈，这个小孩你淡定点啊，这个世界上没有完美的人类，没有完美的父母，没有完美的子女，没有完美的朋友，更没有完美的恋人，也没有完美的小狗，所以你也不用是完美的。这次吸取了经验教训的话，下次好好沟通就是，慢慢成长吧。
